scripts/rook-rbd-pool.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -e
+
+ROOK_VERSION=${ROOK_VERSION:-"v1.1.7"}
+ROOK_DEPLOY_TIMEOUT=${ROOK_DEPLOY_TIMEOUT:-300}
+ROOK_URL="https://raw.githubusercontent.com/rook/rook/${ROOK_VERSION}/cluster/examples/kubernetes/ceph"
+ROOK_RBD_POOL=${ROOK_RBD_POOL:-"newrbdpool"}
+
+function create_pool() {
+    curl -o newpool.yaml "${ROOK_URL}/pool-test.yaml"
+    sed -i "s/replicapool/$ROOK_RBD_POOL/g" newpool.yaml
+	kubectl create -f "./newpool.yaml"
+
+    for ((retry = 0; retry <= ROOK_DEPLOY_TIMEOUT; retry = retry + 5)); do
+		echo "Checking RBD ($ROOK_RBD_POOL) stats... ${retry}s" && sleep 5
+
+		TOOLBOX_POD=$(kubectl -n rook-ceph get pods -l app=rook-ceph-tools -o jsonpath='{.items[0].metadata.name}')
+		TOOLBOX_POD_STATUS=$(kubectl -n rook-ceph get pod "$TOOLBOX_POD" -ojsonpath='{.status.phase}')
+		[[ "$TOOLBOX_POD_STATUS" != "Running" ]] && \
+			{ echo "Toolbox POD ($TOOLBOX_POD) status: [$TOOLBOX_POD_STATUS]"; continue; }
+
+		if kubectl exec -n rook-ceph "$TOOLBOX_POD" -it -- rbd pool stats "$ROOK_RBD_POOL" &>/dev/null; then
+			echo "RBD ($ROOK_RBD_POOL) is successfully created..."
+			break
+		fi
+	done
+
+	if [ "$retry" -gt "$ROOK_DEPLOY_TIMEOUT" ]; then
+		echo "[Timeout] Failed to get RBD pool $ROOK_RBD_POOL stats"
+		exit 1
+	fi
+	echo ""
+}
+
+function delete_pool() {
+	curl -o newpool.yaml "${ROOK_URL}/pool-test.yaml"
+    sed -i "s/replicapool/$ROOK_RBD_POOL/g" newpool.yaml
+	kubectl delete -f "./newpool.yaml"
```

Comment:
we also need to delete newpool.yaml file better to save it in tmp folder

Reply:
`tmp` folder save did not work for me, due to (possible) minikube settings, hence I decided to just dump it in place. If this is fine, we can look at this later.